databricks_cli/unity_catalog/schema_cli.py

```diff
@@ -0,0 +1,146 @@
+              help='Name of the parent catalog for schemas of interest.')
+@click.option('--name-pattern', default=None,
+              help='SQL LIKE pattern that the schema name must match to be in list.')
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def list_schemas_cli(api_client, catalog_name, name_pattern):
+    """
+    List schemas.
+
+    Calls the 'listSchemas' RPC endpoint of the Unity Catalog service.
+    Returns array of SchemaInfos.
+
+    """
+    schemas_json = UnityCatalogApi(api_client).list_schemas(catalog_name, name_pattern)
+    click.echo(mc_pretty_format(schemas_json))
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Get a schema.')
+@click.option('--full-name', required=True,
+              help='Full name (<catalog>.<schema>) of the schema to get.')
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def get_schema_cli(api_client, full_name):
+    """
+    Get a schema.
+
+    Calls the 'getSchema' RPC endpoint of the Unity Catalog service.
+    Returns nothing.
+
+    """
+    schema_json = UnityCatalogApi(api_client).get_schema(full_name)
+    click.echo(mc_pretty_format(schema_json))
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Update a schema.')
+@click.option('--full-name', required=True,
+              help='Full name (<catalog>.<schema>) of the schema to update.')
+@click.option('--json-file', default=None, type=click.Path(),
+              help='File containing JSON request to PATCH.')
+@click.option('--json', default=None, type=JsonClickType(),
+              help=JsonClickType.help('/api/2.0/schemas'))
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def update_schema_cli(api_client, full_name, json_file, json):
+    """
+    Update a schema.
+
+    Calls the 'updateSchema' RPC endpoint of the Unity Catalog service.
+    Returns nothing.
+
+    """
+    json_cli_base(json_file, json,
+                  lambda json: UnityCatalogApi(api_client).update_schema(full_name, json))
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Delete a schema.')
+@click.option('--full-name', required=True,
+              help='Full name (<catalog>.<schema>) of the schema to delete.')
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def delete_schema_cli(api_client, full_name):
+    """
+    Delete a schema.
+
+    Calls the 'deleteSchema' RPC endpoint of the Unity Catalog service.
+    Returns nothing.
+
+    """
+    UnityCatalogApi(api_client).delete_schema(full_name)
+
+
+def register_schema_commands(cmd_group):
+    cmd_group.add_command(create_schema_cli, name='create-schema')
+    cmd_group.add_command(list_schemas_cli, name='list-schemas')
+    cmd_group.add_command(get_schema_cli, name='get-schema')
+    cmd_group.add_command(update_schema_cli, name='update-schema')
+    cmd_group.add_command(delete_schema_cli, name='delete-schema')
```

Comment:
please make command naming consistent with the rest of the CLI (see comments for other command groups)

Reply:
Addressed in #471.